qemu/tests/cfg/netkvm_in_use.cfg

```diff
@@ -9,55 +9,28 @@
     master_images_clone = image1
     remove_image_image1 = yes
     kill_vm_on_error = yes
-    check_guest_bsod = yes
     driver_name = "netkvm"
-    run_bgstress = netperf_stress
-    bg_stress_run_flag = netperf_run
-    netperf_link = netperf-2.7.1.tar.bz2
-    server_path = /var/tmp/
-    client_path = /var/tmp/
-    test_protocols = TCP_STREAM
-    netperf_client = ${vms}
+    hostpassword = redhat
+    netperf_client = ${main_vm}
     netperf_server = localhost
-    netperf_test_duration = 200
-    wait_before_subtest = 100
-    ping_ext_host = "yes"
-    ext_host_get_cmd = "ip route | awk '/default/ { print $3 }'"
-    shell_client = "nc"
-    target_process = netperf.exe
-    variants:
-        - before_bg_test:
-            run_bg_flag = "before_bg_test"
-        - during_bg_test:
-            run_bg_flag = "during_bg_test"
-        - after_bg_test:
-            run_bg_flag = "after_bg_test"
+    netperf_server_link = netperf-2.7.1.tar.bz2
+    server_path = /var/tmp/
+    client_path = ${server_path}
+    netperf_test_duration = 120
+    netperf_para_sessions = 6
+    netperf_output_unit = m
+    netperf_sizes = 1024
+    test_protocol = TCP_STREAM
+    netperf_client_link = "netperf.exe"
+    client_path = "c:\\"
     variants:
         - with_stop_continue:
-            sub_test = stop_continue
-            suppress_exception = no
+            sub_command = stop_continue
```

Comment:
I will prefer to keep the name `sub_test`, it is still a 'test' instead of 'command', if there is name conflict, we might use `netkvm_sub_test`.